django-backend/fecfiler/validation/serializers.py

```diff
@@ -38,6 +38,10 @@ def get_schema_name(self, data):
             :py:function:`fecfile_validate.validate` that will
             match a schema defined in the package
         """
+        request = self.context.get("request", None)
+        if request and request.query_params.get("schema"):
+            self.schema_name = request.query_params.get("schema")
```

Comment:
this got me wondering if setting the schema name would impact subsequent requests.  basically is a single serializer instance used in drf for each request.  I ran a little test printing `schema_name` before and after we set it.  It does not persist across requests.
![image](https://user-images.githubusercontent.com/97105825/169117517-c9c17571-207f-4b96-b7d6-24539721e126.png)

Reply:
so no change required